docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+(try for example ```--processes st,tt``` instead) and to further distinguish between sub-processes
+(e.g. via ```--processses, tt_sl,tt_dl,tt_fh```).
+
+:::{dropdown} Note! Do not add the same dataset via multiple processes!
+At the time of writing this documentation, there is still an issue present that histograms corresponding
+to a dataset can accidentally be used multiple times. For example, when adding ```--processes tt,tt_sl```,
+the events corresponding to the dataset ```tt_sl_powheg``` will be displayed twice in the resulting
+plot.
+:::
+
+
+## Customization of plots
+
+There are many different parameters implemented that allow customizing the style of a plot. A short
+description to all plotting parameters is given in the [Plotting tasks](../task_overview/plotting.md).
+In the following, a few exemplary task calls are given to present the usage of our plotting parameters.
+
+Per default, the PlotVariables1D task creates one plot per variable with all Monte Carlo backgrounds being included
+in a stack and data being shown as separate points. The bottom subplot shows the ratio between signal
+and all processes included in the stack and can be disabled via the ```--skip_ratio``` parameter.
+To change the text next to the label, you can add the ```--cms-label``` parameter.
+To compare shapes of multiple processes, you might want to plot each process separately as one line.
+To achieve this, you can use the ```unstack``` option of the ```--process-settings``` parameter. This
+parameter can also be used to change other attributes of your process instances, such as color, label,
+and the scale. To better compare shapes of processes, we can normalize each line with the
+```--shape-norm``` parameter. Combining all the previously discussed parameters might lead to a task
+call such as
+
+```shell
+law run cf.PlotVariables1D --version v1 --processes tt,st --variables n_jet,jet1_pt \
+    --skip-ratio --shape-norm --cms-label simpw \
+    --process-settings "tt,unstack,color=#e41a1c:st,unstack,label=Single Top"
+```
+
+to produce the following plot:
+
+::::{grid} 1 1 2 2
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__0191de868f__plot__proc_2_a2211e799f__cat_incl__var_jet1_pt__c1.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__0191de868f__plot__proc_2_a2211e799f__cat_incl__var_n_jet__c1.pdf
+:width: 100%
+:::
+::::
+
+
+Parameters that only contain a single value can also be passed via the ```--general-settings```.
+We can also change the y-scale of the plot to a log scale by adding ```--yscale log``` and change some
+properties of specific variables via the ```variable-settings``` parameter. An exemplary task call
+might be
+
+```shell
+law run cf.PlotVariables1D --version v1 --processes tt,st --variables n_jet,jet1_pt \
+    --general-settings "skip_ratio,shape_norm,yscale=log,cms-label=simpw" \
```

Comment:
Stupid question, but why do you have to use the `--general-settings` option in the first place? Can't you just use

```
--skip-ratio --shape-norm --yscale log --cms-label simpw
```

instead?

Reply:
Yes, they are producing the same result, but I wanted to showcase how to use the general_settings.

Reply:
OK, I see your point and generally agree. But as a reader, I would be wondering _why_ I need the general_settings in the first place if I can do the same with the command line options. I think adding some explanation(s) about when to use which of these ways of parsing options would be beneficial

Reply:
added :+1: